winvnc/winvnc/vnckeymap.cpp

```diff
@@ -748,6 +753,43 @@ class Keymapper {
 
         return;
       }
+	  else if (down && (vkCode == VK_HANGEUL))
+	  {
+		  HWND hWnd = GetForegroundWindow();
+		  HWND hIME_Main = ImmGetDefaultIMEWnd(hWnd);
+		  LRESULT MainStatus = ::SendMessage(hIME_Main, WM_IME_CONTROL, 5, 0);
+		  char ch[255];
+		  GetWindowText(hWnd, ch, 254);

```

Comment:
`ch` is not used. Is this some debug aid?

Reply:
It was used temporarily for debugging purposes. 
Delete it and commit again! Thank you for the review 👍